translations/en.json

```diff
@@ -449,6 +471,13 @@
             },
             "header": "Remove {{nodeName}} public link"
         },
+        "revokeInvitationLink": {
+            "body": "By revoking this link, you are blocking access to {{nodeName}} for anyone who tries to use the link to access the file",
```

Comment:
By revoking this link, you are blocking the possibility to create new shares with it. Everyone who has already used the collaboration link will keep the access to the node.